content/english/blog/post-oscar-text-to-speech.md

```diff
@@ -0,0 +1,124 @@
+
+
+
+
+### STEP 1: Deploy the Service
+
+We will use OSCAR-CLI to deploy the use-case service, [this](https://github.com/grycap/oscar/tree/master/examples/text-to-speech) directory contains all the required files. You can open and see all the files:
+  * The Dockerfile to create our [text-to-speech container image](https://github.com/orgs/grycap/packages/container/package/text-to-speech), using Ubuntu 20.04 as base image and installing all the required dependencies.
+  * The code, implemented Python, uses the [Google Speech](https://pypi.org/project/google-speech/) library to convert text to speech.
+  * The user-script that will be executed inside the container when the service is triggered is in the file "script.sh" and it will start the Python program.
+  * Last but not least, the YAML-based Functions Definition File (FDL) provides the definition to create the service via oscar-cli.
+
+
+#### STEP 1.1: Get the YAML file ready
+
+Check in the yaml file that the cluster identifier is defined (OSCAR-CLI has to be configured with the same cluster identifier) and set the "language" environment variable to the language you want to hear the output voice. If you do not know the language code, you can find it [here](https://www.andiamo.co.uk/resources/iso-language-codes/).
+
+```
+functions:
+  oscar:
+  - oscar-cluster:
+      name: text-to-speech
+      memory: 1Gi
+      cpu: '1.0'
+      image: ghcr.io/grycap/text-to-speech
+      script: script.sh
+      log_level: CRITICAL
+      input:
+      - storage_provider: minio
+        path: text-to-speech/input
+      output:
+      - storage_provider: minio
+        path: text-to-speech/output
+      environment: 
+        Variables:
+          language: en
+```
+
+
+
+
+#### STEP 1.2: Use OSCAR-CLI to Deploy the Service
+
+To deploy the service, use the command:
+```sh
+oscar-cli apply tts.yaml
+```
+![03-oscar-apply.png](../../images/blog/post-text-to-speech/03-oscar-apply.png)
+
+
+
+### STEP 2: Verify the Service
+
+After some seconds, the service will be created. The input and output buckets will be automatically created as well. Verify that the service is up and running with the command:
+
+```sh
+oscar-cli service list
+```
+
+![04-oscar-checkServices.png](../../images/blog/post-text-to-speech/04-oscar-checkServices.png)
+
+
+### STEP 3: Invoke the Service Synchronously and Access the Output Files
+
+To run the service synchronously, use the command:
+
+```sh
+oscar-cli service run text-to-speech --text-input "Hello everyone"  --output output.mp3
+```
+
+You also can pass a text file by substituting the flag `--text-input {string}` to `--input {filepath}`
+
+And if you have installed vlc and you want to directly play the result, use this one:
+
+```sh
+oscar-cli service run text-to-speech --text-input "Hello everyone"  --output output.mp3 && vlc output.mp3
+```
+
+![05-oscar-run.png](../../images/blog/post-text-to-speech/05-oscar-run.png)
+
+
+#### STEP 3.1: Asynchronously
```

Comment:
Please, add in this section the commands required to execute asynchrnously using oscar-cli